src/main/scala/org/scalacheck/ScalaCheckFramework.scala

```diff
@@ -81,20 +82,23 @@ private abstract class ScalaCheckRunner extends Runner {
 
     def execute(handler: EventHandler, loggers: Array[Logger],
       continuation: Array[Task] => Unit
-    ): Unit  = continuation(execute(handler,loggers))
+    ): Unit  = continuation(execute(handler,loggers.asInstanceOf[Array[Logger | Null]]).map(_.nn)) // java method array argument contents need to be nullable.
```

Comment:
Pass `Array[T]` as argument to java method that requires an array. Java arrays have type `Array[T | Null]`.